extensions/ql-vscode/src/vscode-tests/cli-integration/run-cli.test.ts

```diff
@@ -52,4 +52,15 @@ describe('Use cli', function() {
     expect(qlpacks['codeql-javascript']).not.to.be.undefined;
     expect(qlpacks['codeql-python']).not.to.be.undefined;
   });
+
+  it('should resolve languages', async function() {
+    skipIfNoCodeQL(this);
+    const languages = await cli.resolveLanguages();
+    // should have a bunch of languages. just check that a few known ones exist
+    expect(languages['cpp']).not.to.be.undefined;
+    expect(languages['csharp']).not.to.be.undefined;
+    expect(languages['java']).not.to.be.undefined;
+    expect(languages['javascript']).not.to.be.undefined;
+    expect(languages['python']).not.to.be.undefined;
```

Comment:
Reading the [Chai docs](https://www.chaijs.com/api/bdd/#method_property), I think there's an equivalent way of writing what we had before without just crashing if the key is missing. Does this work?
```ts
for (const expectedLanguage of ['cpp', 'csharp', 'go', 'java', 'javascript', 'python']) {
  expect(languages).to.have.property(expectedLanguage).that.is.not.undefined;
}
```
I also added Go since we can expect that to exist, might as well test all supported languages.

Reply:
Thank you, that looks nicer! (and also works 😄)

I'll merge this and start looking at more interesting tests in a follow-up PR.